src/pages/RunDetails.js

```diff
@@ -210,14 +210,12 @@ export const RunDetails = ({ submissionId, workflowId }) => {
                   {
                     style: { marginBottom: '1rem' },
                     initialOpenState: true,
-                    title: div({ style: elements.sectionHeader }, [
-                      'Workflow-Level Failures',
-                      h(ClipboardButton, {
-                        text: JSON.stringify(failures, null, 2),
-                        style: { marginLeft: '0.5rem' },
-                        onClick: e => e.stopPropagation() // this stops the collapse when copying
-                      })
-                    ])
+                    title: div({ style: elements.sectionHeader }, 'Workflow-Level Failures'),
+                    afterTitle: h(ClipboardButton, {
+                      text: JSON.stringify(failures, null, 2),
+                      style: { marginLeft: '0.5rem' },
+                      onClick: e => e.stopPropagation() // this stops the collapse when copying
```

Comment:
Didn't expect our conversation on event propagation to be useful so soon

Reply:
One suggestion: Can you add the `e.stopPropagation` call on the `onClick` method in `ClipboardButton` itself? The component should be defined on `common.js`.

If the method call is defined there then you wouldn't have to add the onClick every time you used `ClipboardButton` on the template